datamade/la-metro-councilmatic
# Update advisory committee links, connects #743

## Overview

See title.

### Checklist


## Testing Instructions

 * Run the app: `docker-compose up app`
 * Navigate to the following pages: http://localhost:8001/events and http://localhost:8001/committees . Click the Advisory Committees link and confirm it leads to the requested URL. (You do not need any scraped data in your database to view these pages.)
 
Handles #743
